enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+for all hosts?) explicitly, it would make more sense if we used the information the admin is 
+providing us at install time to correctly configure the cluster as if they had done it themselves, 
+rather than introducing additional sources of (potentially conflicting) configuration 
+information.
+
+## Motivation
+
+See summary above, motivation is to make the install experience clearer with less potential for misconfiguration and conflicting settings.
+
+### Goals
+
+- Post install the cluster can not only pull images from an alternate registry, but imagestreams can import from it, builds can pull from it, and the internal registry can perform pullthrough against it
+- Post install the registry CAs provided by the admin are populated into the configuration api that is intended for this purpose
+
+### Non-Goals
+
+- Refactoring the additionalTrustBundle field into many unique fields for different CA usages
+- Addressing the current proxy CA behavior in which the additionalTrustBundle is sometimes populated into a proxy configuration and other times not (depending on whether a proxy host is supplied)
+- Adding generic cluster configuration fields to the installer that are not required to bootstrap the cluster and ensure consistent use of cluster config apis
+
+
+## Proposal
+
+The installer should populate the 
+[image config resource](https://github.com/openshift/api/blob/master/config/v1/types_image.go) in the same way the 
+[product documentation](https://docs.openshift.com/container-platform/4.5/openshift_images/image-configuration.html#images-configuration-cas_image-configuration) 
+recommends.  The installer 
+can use the value of the additionalTrustBundle field to populate this api.  Similar to how the 
+value is used for the proxy config when a proxy host is supplied, the value can be used to 
+populate the image config resource when a mirror registry is in use.
+
+In addition, the downstream components that potentially communicate with the mirror registry should add logic which confirms they can successfully access the mirror registry.  Specifically, for any registry defined in the image content source policy, the components should be able to access the registry using the supplied CAs.  If they cannot, they should mark themselves degraded.
+
+Finally, if possible, the MCO mechanism that injects CAs into the nodes directly should be disabled when the image config api is populated, to ensure we do not have conflicting values being placed on the nodes.  The MCO mechanism should be a pure bootstrapping mechanism.  (That may already be the case today, I am not certain how that mechanism is implemented).
+
+### User Stories
+
+#### Story 1
+
+As an administrator installing a new cluster, I do not want to have to take additional actions on day 2 of my cluster installation or risk forgetting to do so only to have surprising failures later when the cluster is in use.  
```

Comment:
That user story means everything needs to happen day-1 and I am not in favor of that philosophy because it moves the installer to 3.x where the number of configurations for day-1 ended up being too large for us safely support.

Can you provide a little more specific user story.

Reply:
will do